dbt/adapters/default.py

```diff
@@ -522,7 +540,7 @@ def add_query(cls, profile, sql, model_name=None, auto_begin=True):
             pre = time.time()
 
             cursor = connection.get('handle').cursor()
-            cursor.execute(sql)
+            cursor.execute(sql, (bindings or ()))
```

Comment:
@cmcarthur this failed to run for me on Redshift. If `bindings` is None, then this expression evaluates to an empty tuple. I think we want that to be `None`, right?  http://initd.org/psycopg/docs/cursor.html#cursor.execute